### PR TITLE
Move sig-scale meetings to 10:30 PT

### PR DESCRIPTION
**Pull request by mm4tt**
We decided that it'd be better than having the meeting time in UTC. Using UTC results in having the meeting at one time during half of the year and then at other time during another half. It makes hard as it requires re-organizing your calendar every 6 months. Using PT will keep the time the same for America. In Europe, we'll have two meetings at different time but we're fine with that.

### CODE REVIEW

File: sig-scalability/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ SIG Scalability is responsible for defining and driving scalability goals for Ku
 The [charter](charter.md) defines the scope and governance of the Scalability Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [Thursdays at 17:30 UTC](https://zoom.us/j/94252896018?pwd=cTlMMlBoTHZqUEdjRm9VY2NWNUg5dz09) (bi-weekly ([upcoming meeting dates](#upcoming-meeting-dates))). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=UTC).
+* Regular SIG Meeting: [Thursdays at 10:30 PT](https://zoom.us/j/94252896018?pwd=cTlMMlBoTHZqUEdjRm9VY2NWNUg5dz09) (bi-weekly ([upcoming meeting dates](#upcoming-meeting-dates))). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:30AM&tz=California).
```
Comment (wojtek-t) [inline sig-scalability/README.md:16]: You need to change that in sigs.yaml - this part is autogerated from that file.

Comment (mm4tt) [reply sig-scalability/README.md]: Yeah, I forgot...

**Pull request merged**